File: peek_plugin_docdb/plugin-module/_private/tuples/AddIntValueActionTuple.ts
 import {addTupleType, Tuple, TupleActionABC} from "@synerty/vortexjs";
import {docDbTuplePrefix} from "../PluginNames";

@addTupleType
export class AddIntValueActionTuple extends TupleActionABC {
    static readonly tupleName = docDbTuplePrefix + "AddIntValueActionTuple";

    documentId: number;
    offset: number;

    constructor() {
        super(AddIntValueActionTuple.tupleName)
    }
}

File: peek_plugin_docdb/_private/tuples/AddIntValueActionTuple.py
from vortex.Tuple import addTupleType, TupleField
from vortex.TupleAction import TupleActionABC

from peek_plugin_docdb._private.PluginNames import docDbTuplePrefix


@addTupleType
class AddIntValueActionTuple(TupleActionABC):
    __tupleType__ = docDbTuplePrefix + "AddIntValueActionTuple"

    documentId = TupleField()
    offset = TupleField()
